src/controllers/capacity/index.ts

```diff
@@ -138,6 +138,16 @@ export class CapacityController extends Controller {
     if (!creation) throw new NotFound('creation')
     return creation
   }
+
+  @Response<NotFound>(404, 'Items not found.')
+  @SuccessResponse('200')
+  @Get('{capacityId}/creation/')
+  public async getTransactionsFromCapacity(@Path() capacityId: UUID): Promise<TransactionResponse[]> {
+    const capacity = await this.db.getTransactionsFromCapacityID(capacityId)
+    if (capacity.length === 0) throw new NotFound('capacity')
```

Comment:
a length of `0` is ok because there might be no creation transactions for the capacity, you would return `[]`.

However it should `404 - NotFound` if the capacity doesn't exist at all